src/engines/nwn2/roster.cpp

```diff
@@ -39,6 +39,42 @@ Roster::Roster() {
 Roster::~Roster() {
 }
 
+/** Return the roster name of the first member. */
+Common::UString Roster::getFirstRosterMember() {
+	Common::UString name = "";
+	_lastRetrieved = 0;
+
+	// Set to the first member's name
+	if (!_members.empty())
+		name = (_members.begin())->rosterName;
+
+	return name;
+}
+
+/** Return the roster name of the next member. */
+Common::UString Roster::getNextRosterMember() {
+	Common::UString name = "";
+
+	if (_members.empty()) {
+		_lastRetrieved = 0;
```

Comment:
I'd just do an early return there then, so that the "meat" of the method isn't in that else if branch.